runtime/kubernetes/service.go

```diff
@@ -66,6 +66,9 @@ func newService(s *runtime.Service, c runtime.CreateOptions) *service {
 			kdeploy.Spec.Template.PodSpec.Containers[i].Command = []string{}
 			kdeploy.Spec.Template.PodSpec.Containers[i].Args = []string{}
 		}
+	} else {
+		// use the default image (micro/cell:go), passing the source as the first argument
+		kdeploy.Spec.Template.PodSpec.Containers[0].Args = []string{s.Source}
```

Comment:
is this going to cause problems? The idea was to remove any clever logic from here and keep it in the CLI since that has knowledge of any requirements.